lib/broccoli/ember-app.js

```diff
@@ -1227,11 +1370,21 @@ EmberApp.prototype.javascript = function() {
       }),
       merge(this._prunedBabelOptions())
     );
+
+    // fastboot app may not be present for vanilla ember apps
+    fastbootAppJs = new Babel(
+      new Funnel(fastbootApplicationJs, {
+        destDir: this.options.outputPaths.fastboot.app,
+        allowEmpty: true,
+        annotation: 'Funnel: Fastboot App JS files'
+      }),
+      merge(this._prunedBabelOptions())
+    );
   }
 
   appJs = mergeTrees([
     appJs,
-    this._processedEmberCLITree()
+    this._processedEmberCLITree() // why call this again when called in appAndDependencies() ?
```

Comment:
Because of the overwrite behavior. Should generally be safe, but could theoretically result in bonus assets.